.github/workflows/check-pr-title.yaml

```
@@ -0,0 +1,14 @@
+name: pr title check
+on:
+  pull_request:
+    types: [opened, edited, ready_for_review, reopened]
+jobs:
+  lint-pr-title:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.draft == false && github.event.pull_request.state == 'open'
+    permissions:
+      pull-requests: read
+    steps:
+      - uses: Slashgear/action-check-pr-title@v4.0.0
+        with:
+          regexp: '\[[A-Z]+\] .+'
```

```suggestion
          regexp: '\[[A-Z]+\] .+\.'
```
Les titres des PR doivent normalement contenir un `.` à la fin. Peut-être peut-on l'ajouter facilement ?

---

Je ne suis pas pour car ce point n'apporte rien IMO (perso je l'ai jamais mis)

---

Le point est présent dans le Contributing.md, et permet une cohérence visuelle entre les PR (certaines finissent avec rien, d'autres avec des parenthèses liés au numéro de ticket). 
IMO, ca reste une bonne pratique peu coûteuse.

---

Je vois dans les PRs ouvertes en ce moment qu'il y en a beaucoup qui n'ont pas de point. Je ne comprend pas l'argument de la cohérence visuelle ?

Je préfère ne pas trancher le débat dans cette PR si l'avis est partagé en tout cas, donc de laisser libre ou non de ne le mettre. Ça n'empêchera pas d'avoir ce débat dans une autre PR comme pour d'autres contraintes que l'on souhaiterait ajouter. WDYT ?

---

Je suis également pour mettre un point à la fin, je ne le fais pas pour autant à chaque fois. Pour moi un Linter sert à s'assurer/imposer une rigueur sur un format donc ça collerait. 
Après je serais d'accord que cette PR n'a pas pour vocation d'imposer ou affirmer une nouvelle règle mais de vérifier une règle déjà bien partagée donc on pourrait passer la PR sans ça.

---

Pour la cohérence, je pensais en particulier au Changelog: https://github.com/1024pix/pix/blob/dev/CHANGELOG.md

---

Ah ok merci pour la précision :)

On pourrait l'ajouter automatiquement dans la génération du changelog car [c'est nous qui l'implémentons](https://github.com/1024pix/pix-bot/blob/c61bb7aa8f6705331016c26913c2d38bf5378007/test/unit/common/services/changelog_test.js#L4) ! 😄 

On en rediscute dans une autre PR 😉

---

Point ajouté automatiquement au changelog avec https://github.com/1024pix/pix-bot/pull/256